docs/sdks/uid2-sdk-ref-mobile-gma.md

```diff
@@ -0,0 +1,99 @@
+# Developer Guide for UID2 GMA Plugin (iOS and Android)
+
+As of 2023-07-15 plugin version:
+* iOS V 0.2.0
+* Android V 0.4.0
+
+## Introduction
+
+The UID2 GMA Plugin is pre-release software and should be used as such.
+
+The UID2 GMA Plugin enables apps that use Google Mobile Ad (GMA) SDK to
+include UID2 data in their ad requests. This is done automatically, with
+no direct coding needed from the app developer, by:
+
+-   The Google Mobile Ad SDK
+
+-   The UID2 SDK
+
+-   The UID2 GMA Plugin
+
+## iOS 
+
+Refer to the following sections for iOS:
+
+-   Requirements
+
+-   Installation
+
+### Requirements 
+
+Install the following:
+
+-   UID2 iOS SDK v0.2.0
+-   [Google Mobile Ad SDK v10.7.0+](https://developers.google.com/admob/ios/rel-notes)
```

Comment:
@bleege should I keep this as 10.7.0+ or just 10.7.0?

Reply:
Let's leave it as `v10.7.0+` as it helps communicate that this is a minimum version and not a "only this version can be used" requirement.